src/components/Calculator.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * /* eslint-disable react/prefer-stateless-function
+ *
+ * @format
+ */
```

Comment:
> - [ ] Hello @Sami-Ullah-Tufail, It's not good practice to disable linters because linters help maintain code quality and catch potential issues. Instead of disabling them, it's recommended to address and fix any linting errors or warnings to ensure your code adheres to best practices and conventions.👍